src/main/java/org/sasanlabs/service/vulnerability/ssrf/SSRFVulnerability.java

```diff
@@ -1,52 +1,221 @@
 package org.sasanlabs.service.vulnerability.ssrf;
 
-import com.nimbusds.jose.util.StandardCharset;
-import java.io.InputStream;
+import java.io.BufferedReader;
```

Comment:
The file permissions should remain same. please revert the permissions to 100644.